performance_test/CMakeLists.txt

```diff
@@ -120,6 +120,10 @@ if(PERFORMANCE_TEST_ODB_FOR_SQL_ENABLED)
   list(APPEND CMAKE_MODULE_PATH "${CMAKE_SOURCE_DIR}/cmake/Modules")
   find_package(ODB REQUIRED COMPONENTS sqlite OPTIONAL_COMPONENTS mysql pgsql)
   include(${ODB_USE_FILE})
+
```

Comment:
After find_package, the following variables for finding ODB are set:

  * ODB_LIBODB_INCLUDE_DIRS
  * ODB_LIBODB_LIBRARIES
  * ODB_EXECUTABLE
  * ODB_INCLUDE_DIRS

ODB_LIBODB_LIBRARIES is something like /opt/odb/lib/libodb.so.  I strip off the file component to get the lib directory